crypto/keyring/keyring.go

```diff
@@ -483,11 +483,11 @@ func (ks keystore) Delete(uid string) error {
 func (ks keystore) KeyByAddress(address sdk.Address) (Info, error) {
 	ik, err := ks.db.Get(addrHexKeyAsString(address))
 	if err != nil {
-		return nil, wrapKeyNotFound(err, fmt.Sprint("key with address", address, "not found"))
+		return nil, wrapKeyNotFound(err, fmt.Sprint("key with address ", address, " not found"))
```

Comment:
I feel we can use `addrHexKeyAsString(address)` instead of `address` on the errors

Reply:
+1 for more user-readble errors! What do you think of printing the bech32 address (`address.String()`) instead of it's hex encoding?